content/app/app-dev-course/modul7/_index.en.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Module 7
+description: Standalone extensions of the app
+linktitle: Module 7
+tags: [apps, training ]
+weight: 20
+---
+
+{{% notice warning %}}
+NOTE! This module is still under development. Come back later and it will probably be even better!
+{{% /notice %}}
+
+In this module there are a collection of standalone extensions of the application. They do not require being solved in order.
+
+
+**Topics covered in this module:**
+- Summary page
+- Statless application
+- Variables in text
+- Use of external API
+- Configuration of message box
+- Presentation texts
+
+### Tasks
+
+{{% expandlarge id="Summary page" header="Summary page" %}}
+### Requirements from the muncipality
+The muncipality of Sogndal are experiencing an issue where some submissions contain incorrect information and typos that create unnecessary work for the caseworkers.
+To avoid people submitting incorrect information, the muncipality wishes that the user is presented with a summary page displaying the filled out information.
```

Comment:
```suggestion
The municipality of Sogndal are experiencing an issue where some submissions contain incorrect information and typos that create unnecessary work for the caseworkers.
To avoid people submitting incorrect information, the municipality wishes that the user is presented with a summary page displaying the filled out information.
```